*** wp-includes/formatting.php.orig	Sat May  9 03:03:02 2026
--- wp-includes/formatting.php	Sat May  9 03:03:47 2026
***************
*** 4489,4502 ****
  function esc_url( $url, $protocols = null, $_context = 'display' ) {
  	$original_url = $url;
  
! 	if ( '' === $url ) {
  		return $url;
  	}
  
  	$url = str_replace( ' ', '%20', ltrim( $url ) );
  	$url = preg_replace( '|[^a-z0-9-~+_.?#=!&;,/:%@$\|*\'()\[\]\\x80-\\xff]|i', '', $url );
  
! 	if ( '' === $url ) {
  		return $url;
  	}
  
--- 4489,4504 ----
  function esc_url( $url, $protocols = null, $_context = 'display' ) {
  	$original_url = $url;
  
! 	/* We use strcmp instead of ===, therefore the wpgarlic mocked equality won't be used */
! 	if (!strcmp('', $url)) {
  		return $url;
  	}
  
  	$url = str_replace( ' ', '%20', ltrim( $url ) );
  	$url = preg_replace( '|[^a-z0-9-~+_.?#=!&;,/:%@$\|*\'()\[\]\\x80-\\xff]|i', '', $url );
  
! 	/* We use strcmp instead of ===, therefore the wpgarlic mocked equality won't be used */
! 	if (!strcmp('', $url)) {
  		return $url;
  	}
  
