*** /var/www/html/wp-includes/functions.php.orig	Sat May  9 03:00:20 2026
--- /var/www/html/wp-includes/functions.php	Sat May  9 03:00:54 2026
***************
*** 678,684 ****
  		return false;
  	}
  	$data = trim( $data );
! 	if ( 'N;' === $data ) {
  		return true;
  	}
  	if ( strlen( $data ) < 4 ) {
--- 678,685 ----
  		return false;
  	}
  	$data = trim( $data );
! 	/* We use strcmp instead of ===, therefore the wpgarlic mocked equality won't be used */
!         if ( !strcmp('N;', $data) ) {
  		return true;
  	}
  	if ( strlen( $data ) < 4 ) {
