*** wp-load.php.orig	Sat May  9 02:56:44 2026
--- wp-load.php	Sat May  9 02:57:06 2026
***************
*** 66,74 ****
  	// Check for the required PHP version and for the MySQL extension or a database drop-in.
  	wp_check_php_mysql_versions();
  
- 	// Standardize $_SERVER variables across setups.
- 	wp_fix_server_vars();
- 
  	define( 'WP_CONTENT_DIR', ABSPATH . 'wp-content' );
  	require_once ABSPATH . WPINC . '/functions.php';
  
--- 66,71 ----
